py/obj.c

```diff
@@ -331,6 +331,25 @@ bool mp_obj_get_int_maybe(mp_const_obj_t arg, mp_int_t *value) {
     return true;
 }
 
+int64_t mp_obj_get_ll_int(mp_const_obj_t arg_in) {
+    if (mp_obj_is_small_int(arg_in)) {
+        return MP_OBJ_SMALL_INT_VALUE(arg_in);
+    } else if (mp_obj_is_type(arg_in, &mp_type_int)) {
+        mp_obj_int_t *arg = arg_in;
```

Comment:
This line causes almost every build to fail. It should most likely be:
`        mp_const_obj_int_t *arg = arg_in;`

Reply:
Revert **mp_const_obj_t** to **mp_obj_t**
https://github.com/micropython/micropython/pull/6639/commits/f4bf62522d43a4c699759b191ee0d199a401ec1d